lib/tasks/publishing_api.rake

```diff
@@ -241,9 +241,14 @@ namespace :publishing_api do
   end
 
   desc "Republish all HTML attachments to the Publishing API"
-  task republish_html_attachments: :environment do
-    HtmlAttachment.where(attachable_type: "Edition", attachable_id: Edition.publicly_visible.select("id")).each do |a|
-      Whitehall::PublishingApi.bulk_republish_async(a)
+  task :republish_html_attachments, [:state] => :environment do |_, args|
+    args.with_defaults(state: "withdrawn")
+    publications = Publications.where(state: args[:state])
+    publications.find_each do |publication|
+      if !publication.html_attachments.nil? || publication.html_attachments.count.positive?
```

Comment:
I imagine you can do this in one conditional? `publication.html_attachments&.count&.positive?`

Though I do wonder if this conditional could be done as part of the Publications query to only pull out publications with HTML attachments.